examples/java_postgresql_customvar/README.md

```diff
@@ -202,11 +202,11 @@ spec:
     id: postgresDB
   customEnvVar:
     - name: JDBC_URL
-      value: 'jdbc:postgresql://{{ postgresDB.status.dbConnectionIP }}:{{ .postgresDB.status.dbConnectionPort }}/{{ postgresDB.status.dbName }}'
+      value: 'jdbc:postgresql://{{ .postgresDB.status.dbConnectionIP }}:{{ .postgresDB.status.dbConnectionPort }}/{{ .postgresDB.status.dbName }}'
     - name: DB_USER
-      value: '{{ postgresDB.status.dbConfigMap.username }}'
+      value: '{{ .postgresDB.status.dbCredentials.user }}'
```

Comment:
Would be good to link the corresponding changes in the db operator as well in the PR description.